docs/msbuild/customize-your-build.md

```diff
@@ -15,11 +15,21 @@ ms.workload:
   - "multiple"
 ---
 # Customize your build
+
+MSBuild projects that use the standard build process (importing `Microsoft.Common.props` and `Microsoft.Common.targets`) have several extensibility hooks that can be used to customize your build process.
+
+## Adding arguments to command-line MSBuild invocations for your project
+
+A `Directory.Build.rsp` file in your source directory will be applied to command-line builds of your project. For details, see [MSBuild Response Files](../msbuild/msbuild-response-files.md#directorybuildrsp).
+
+## Directory.Build.props and Directory.Build.targets
+
 In versions of MSBuild prior to version 15, if you wanted to provide a new, custom property to projects in your solution, you had to manually add a reference to that property to every project file in the solution. Or, you had to define the property in a *.props* file and then explicitly import the *.props* file in every project in the solution, among other things.
 
 However, now you can add a new property to every project in one step by defining it in a single file called *Directory.Build.props* at the root of your repo. When MSBuild runs, *Microsoft.Common.props* searches your directory structure for the *Directory.Build.props* file (and *Microsoft.Common.targets* looks for *Directory.Build.targets*). If it finds one, it imports the property. *Directory.Build.props* is a user-defined file that provides customizations to projects under a directory.
```

Comment:
Perhaps the word repo is confusing here since its actually not really connected to source control, can it be something like "root folder" or "base folder for your code" or something? Come to think of it, what folder is it actually looking at if I run the build from another folder? I suppose its the same folder as the project/solution file is in?

Reply:
Clearer after edit?